### ISSUE TITLE
Regarding random effects in predictions

### ISSUE THREAD

**Issue by nisargshah1999** [junyanz/pytorch-CycleGAN-and-pix2pix#1267]
Hi,

I am training a UNet based model(no normalization in generator) as a backbone to pix2pix. While testing the predictions are getting some random effects of noise like shown below.
![10024_fake_B](https://user-images.githubusercontent.com/52718252/113522650-ffedd180-95bf-11eb-83c8-6bcea16ed413.png)
I would be obliged if someone could help me on how to solve it.

**Comment by synthetica3d** [comment]
Hi, did you try using instance norm instead of batch norm?
